From 1bfd5fb51cee008c4ce75cff987bb3c0f32bc7da Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Tue, 20 Apr 2021 06:35:24 +0000
Subject: [PATCH 36/40] WIP: ARM: dts: backport missing gpio binding for WeTek
 Hub/Play2

This change has not been sent upstream yet.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm/dts/meson-gxbb-wetek-hub-u-boot.dtsi   | 1 +
 arch/arm/dts/meson-gxbb-wetek-play2-u-boot.dtsi | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/dts/meson-gxbb-wetek-hub-u-boot.dtsi b/arch/arm/dts/meson-gxbb-wetek-hub-u-boot.dtsi
index 2a245bbe7f..4ddadaf65a 100644
--- a/arch/arm/dts/meson-gxbb-wetek-hub-u-boot.dtsi
+++ b/arch/arm/dts/meson-gxbb-wetek-hub-u-boot.dtsi
@@ -5,6 +5,7 @@
  */
 
 #include "meson-gx-u-boot.dtsi"
+#include <dt-bindings/gpio/gpio.h>
 
 &ethmac {
 	snps,reset-gpio = <&gpio GPIOZ_14 0>;
diff --git a/arch/arm/dts/meson-gxbb-wetek-play2-u-boot.dtsi b/arch/arm/dts/meson-gxbb-wetek-play2-u-boot.dtsi
index 2a245bbe7f..4ddadaf65a 100644
--- a/arch/arm/dts/meson-gxbb-wetek-play2-u-boot.dtsi
+++ b/arch/arm/dts/meson-gxbb-wetek-play2-u-boot.dtsi
@@ -5,6 +5,7 @@
  */
 
 #include "meson-gx-u-boot.dtsi"
+#include <dt-bindings/gpio/gpio.h>
 
 &ethmac {
 	snps,reset-gpio = <&gpio GPIOZ_14 0>;
-- 
2.17.1

